Gemfile

```diff
@@ -15,10 +15,10 @@ gem 'listen'
 gem 'activejob'
 
 # Add support for refinerycms-acts-as-indexed
-gem 'refinerycms-acts-as-indexed', ['~> 3.0', '>= 3.0.0']
+gem 'refinerycms-acts-as-indexed', ['~> 3.0', '>= 3.0.0'], require: 'refinery/acts_as_indexed'
```

Comment:
still wishing there is a way we can get this without the `require`. In this case I guess the gem should be called `refinerycms-acts_as_indexed`

Reply:
It could help but there will always be a problem on the `cms` part

Reply:
not if it's `lib/refinerycms/acts_as_indexed.rb`

Maybe for now we can use `lib/refinerycms/acts/as/indexed.rb`? Haven't tried it yet